### ISSUE TITLE
"No QCoreApplication instance found. Application patches not applied."

### ISSUE THREAD

**Issue by mpeters2** [spyder-ide/spyder#15319]
### Issue Report Checklist

* [Y ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [Y ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [Y ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [Y ] Tried basic troubleshooting (if a bug/error)
    * [Y] Restarted Spyder
    * [Y ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [Y ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

"No QCoreApplication instance found." error when launching from Anaconda Navigator (see pasted error below).  Spyder was running slower, and slower, so I decided to quit and relaunch through Anaconda.  Now it gives me this error and quits.

### What steps reproduce the problem?

1. launching Spyder 5.0.0 from Anaconda
2. spyder --reset" will temporarilly fix the problem.  But when I tweak the preferences (turn off linting, light theme) and spyder restarts, it gives the same crash message
3. 

### Paste Traceback/Error Below (if applicable)

```python-traceback
No QCoreApplication instance found. Application patches not applied. You have to call load_stylesheet function after instantiation of QApplication to take effect. 
Pruned plugin: outline_explorer
Traceback (most recent call last):
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 2146, in create_window
main.setup()
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 847, in setup
plugin_instance = plugin_class(self)
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/plugins/editor/plugin.py", line 196, in __init__
self._setup()
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/plugins/base.py", line 415, in _setup
self._plugin_actions = self.get_plugin_actions() + [MENU_SEPARATOR,
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/plugins/editor/plugin.py", line 493, in get_plugin_actions
name="New file", add_shortcut_to_tip=True)
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/api/plugins.py", line 358, in register_shortcut
add_shortcut_to_tip)
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/plugins/base.py", line 439, in _register_shortcut
self.CONF_SECTION)
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 1403, in register_shortcut
self.shortcuts.register_shortcut(
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 1164, in __getattr__
return super().__getattr__(attr)
AttributeError: 'MainWindow' object has no attribute 'shortcuts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/Applications/anaconda3/bin/spyder", line 11, in 
sys.exit(main())
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/start.py", line 214, in main
mainwindow.main(options, args)
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 2293, in main
mainwindow = create_window(app, splash, options, args)
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 2148, in create_window
if main.console is not None:
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 1159, in __getattr__
return self.get_plugin(self._INTERNAL_PLUGINS_MAPPING[attr])
File "/Applications/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 168, in get_plugin
raise SpyderAPIError('Plugin "{}" not found!'.format(plugin_name))
spyder.api.exceptions.SpyderAPIError: Plugin "internal_console" not found!


```

## Versions

* Spyder version: 5.0.0
* Python version: 3.7.4
* Qt version:  5.9.7
* PyQt version: 5.9.2
* Operating System name/version: OsX Catalina 10.15.7

### Dependencies

```
PASTE DEPENDENCIES HERE

Spyder crashes, so I can't copy the dependencies.
```

**Comment by mpeters2** [comment]
Update: turning off linting seems to be what causes the crash after restarting Spyder (I was doing "spyder --reset" between attempts to track down the preferences change that was leading to crashes).

**Comment by juanis2112** [comment]
Hi @rcrisanti. Thanks for reporting. This is a duplicate of #15177 and was solved in our latest release 5.0.1. I recommend waiting for it to be available in the next days in Anaconda, but if you want you can download it with our installers https://www.spyder-ide.org/#section-download

**Comment by jorgenova** [comment]
Hello, @juanis2112 , how are you?

U said that the issus was solved in the release 5.0.1, but I can´t install this release yet with conda.

When It will be available?

**Comment by ccordoba12** [comment]
We don't know @jorgenova. Unfortunately, we can't decide when new versions are uploaded to Anaconda, sorry.
